README.md

```diff
@@ -10,11 +14,10 @@ on: [ push ]
 
 jobs:
   ziti-action:
-    # Currently only runs on macos
-    runs-on: macos-latest
+    runs-on: ubuntu-latest
     name: Ziti Webhook Action
     steps:
-    - uses: openziti/ziti-webhook-action@main
+    - uses: openziti/ziti-webhook-action@v2
```

Comment:
can't we leave this as 'main'? you'll fix it eventually right?

Reply:
`main` is not recommended for a litany of reasons. Is there any particular reason to use it?